SuperBuild/External_LibArchive.cmake

```
@@ -2,7 +2,7 @@
 set(proj LibArchive)
 
 # Set dependency list
-set(${proj}_DEPENDENCIES "zlib")
+set(${proj}_DEPENDENCIES "zlib" "RapidJSON")
```

```suggestion
set(${proj}_DEPENDENCIES "zlib")
```

This was likely introduced by error as `LibArchive` does not seem to depend on RapidJSON